### ISSUE TITLE
Neoline extension breaks Google maps.

### ISSUE THREAD

**Issue by FrankM0** [NeoNextClub/neoline#133]
As readable in the many https://chromewebstore.google.com/detail/neoline/cphhlgmgameodnhkjdmkpanlelnlohao/reviews extension reviews: the Chrome extension breaks Google Maps, when you disable the extension Maps works as expected.

**Comment by haiwulei** [comment]
The latest release, version 4.0.6 of the NeoLine extension, has resolved this issue. Kindly update your extension.

**Comment by FrankM0** [comment]
I'm afraid it doesn't, just tested with version 4.0.6 and Google Maps doesn't work.
As soon as i disable Neoline Google Maps works and loads its menu structure.

Have you tried using Google Maps with Neoline enabled?

**Comment by haiwulei** [comment]
Yes, we've run numerous tests and found that the Google Maps website works properly on version 4.0.6. If you're still experiencing issues, please provide us with more detailed information, such as a video.
